### DISCUSSION TITLE
Easily missed initial conditions

### DISCUSSION THREAD

**Discussion by mattldawson** [NCAR/micm#568] [Ideas]
Mechanisms that don't include `TroeRateConstant` or `ChemicalActivationRateConstant` don't require that the `Conditions.air_density_` parameter be set. But, there is no check that the air density is set by the user when these rate constant types are included.

Should we modify this behavior somehow to avoid unexpected results?

Is it even reasonable to do a simulation for a non-ideal gas? Or are the rate constant algorithms themselves derived assuming an ideal gas?

**Comment by K20shores** [comment]
This always gets calculated now, I believe
